test/foo/lib/config/config.yaml

```diff
@@ -0,0 +1,3 @@
+VAR0: fizz
```

Comment:
It would be nicer to create file in tests themselves. Splitting requires looking into multiple places, while you would like a test to be easy to read.
https://docs.python.org/3/library/tempfile.html